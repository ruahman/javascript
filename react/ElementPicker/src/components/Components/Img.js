import React from "react";

const Img = ({ values }) => {
  return <img src={values.src} alt={values.alt}></img>;
};

export default Img;
