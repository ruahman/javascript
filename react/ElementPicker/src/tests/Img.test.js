import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Img from '../components/Components/Img.js';

describe('Img', () => {
  it('works ok', () => {
    const component = shallow((
      <Img
        values={{
          alt: 'Cool image',
          src: 'http://lorempixel.com/400/200',
        }}
      />
    ));

    expect(toJson(component)).toMatchSnapshot();
  });
});
