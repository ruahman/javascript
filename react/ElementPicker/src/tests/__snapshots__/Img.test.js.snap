// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Img works ok 1`] = `
<img
  alt="Cool image"
  src="http://lorempixel.com/400/200"
/>
`;
